cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -720,6 +720,48 @@
         ]
       }
     },
+    "/core/packages/v1alpha1/repositories/c/{context.cluster}/permissions": {
```

Comment:
Just FTR, we should also update the `openapi.yaml` file when upgrading the API. 
See https://github.com/vmware-tanzu/kubeapps/pull/5610